source/Playnite/Database/GameDatabase.cs

```diff
@@ -1310,6 +1313,37 @@ bool updateCompletionStatus(Game game, CompletionStatusSettings settings)
             }
         }
 
+        public List<FilterPreset> GetSortedFilterPresets(bool getClone)

```

Comment:
I don't think this getClone is a good idea if you need it just for some specific unit test purpose, it should be handled in the unit test.

Reply:
Parameter removed in https://github.com/JosefNemec/Playnite/pull/3258/commits/4d9ec25d9bf201e2aece128f6c87cc257eda7b60